### ISSUE TITLE
site goal fails on spotbugs branch & oraclejdk8

### ISSUE THREAD

**Issue by KengoTODA** [spotbugs/spotbugs-maven-plugin#40]
Example: https://travis-ci.org/spotbugs/spotbugs-maven-plugin/jobs/361014422
This problem is maybe related with https://github.com/jeremylong/DependencyCheck/issues/523

**Comment by hazendaz** [comment]
I had to fix a few things recently elsewhere due to nve database moved.  I also think Jeremy long corrected on version 3.1.2.  We should be able to get back working completely today.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by KengoTODA** [comment]
Site goal has been stabilized, but deploy phase failed to rsync. I'll propose another PR to fix.

```
rsync: change_dir "/home/travis/build/spotbugs/spotbugs-maven-plugin/home/travis/build/spotbugs/spotbugs-maven-plugin/target/site" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1183) [sender=3.1.0]
Could not copy /home/travis/build/spotbugs/spotbugs-maven-plugin/home/travis/build/spotbugs/spotbugs-maven-plugin/target/site.
failed to deploy
```
